common/app/model/meta.scala

```diff
@@ -723,3 +724,17 @@ object Tags {
     Tags(apiContent.tags.toList map { Tag.make(_) })
   }
 }
+
+object AdUnitMaker extends Logging {
+
+  val knownBadAdUnits = Seq("article")
+
+  def make(pageId: String, adUnitSuffix: String): String = {
+    if (knownBadAdUnits.contains(adUnitSuffix)) {
```

Comment:
It would need to be an exact match as it stands so no.  It might be better if it were a regex match though.
Now we know what the problem is, pages in the global section would make good test cases.  I'll merge it when I have time to do a bit more testing.